packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -262,6 +262,19 @@ const String _kColorForegroundWarning = 'Cannot provide both a color and a foreg
 /// If all custom fallback font families are exhausted and no match was found
 /// or no custom fallback was provided, the platform font fallback will be used.
 ///
+/// ### Inconsistent platform fonts
+///
+/// Since Flutter's font discovery for default fonts depends on the fonts present
+/// on the device, it is not safe to assume all default fonts will be available or
+/// consistent across devices.
+///
+/// A known example of this is that Samsung devices ship with a CJK font that has
+/// smaller line spacing than the Android default. This results in Samsung devices
+/// displaying more tightly spaced text than on Android when no custom font is specified.
```

Comment:
.. than on other Android devices ... ?

Isn't Samsung also running Android?

Reply:
Yes, but samsung decided to ship *2* CJK fonts on their devices: SECCJK-Regular.ttc and NotoSansCJK-Regular.ttc (android def).

Reply:
The default CJK font then resolves to their custom one instead of noto, which I don't think they could remove as too many apps would break.

Reply:
Oops misread your comment, Yes, it should be *other* android devices

Reply:
😄 👍